apps.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
```

Comment:
We don't need this just yet (until you get to a post) so this can be removed for now